src/persist-types/src/stats.proto

```diff
@@ -80,10 +80,11 @@ message ProtoJsonStats {
         google.protobuf.Empty json_nulls = 3;
         ProtoPrimitiveStats bools = 4;
         ProtoPrimitiveStats strings = 5;
-        ProtoPrimitiveStats numerics = 6;
+        ProtoPrimitiveBytesStats numerics = 9;
         google.protobuf.Empty lists = 7;
         ProtoJsonMapStats maps = 8;
     }
+    reserved 6;
```

Comment:
if we've written something with id 6 and then read it in with this code, wont the `None => return Err(TryFromProtoError::missing_field("ProtoJsonStats::values"))` line eventually bubble up as a panic?

Reply:
Excellent catch! I've switched that line to fill in `JsonStats::Mixed`, since that is a safe default for the case where we can't say anything more precise about a particular value.